InventoryInStorePickupFrontend/composer.json

```diff
@@ -6,6 +6,8 @@
         "magento/framework": "*",
         "magento/module-checkout": "*",
         "magento/module-inventory-in-store-pickup-api": "*",
+        "magento/module-inventory-in-store-pickup-sales-api": "*",
+        "magento/module-sales": "*",
```

Comment:
Dependency should be resolved by view model usage.

Reply:
Done.